### PR TITLE
Update k6 logo

### PR DESCRIPTION
**Pull request by mstoykov**
## What?

image
to
image

## Why?

Updating the banner logo. 

Terminal colors are user configurable so on my terminal "yellow" is ... green :P 

## Checklist

- [ ] I have performed a self-review of my code.
- [ ] I have added tests for my changes.
- [ ] I have run linter locally (`make lint`) and all checks pass.
- [ ] I have run tests locally (`make tests`) and all tests pass.
- [ ] I have commented on my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Review by olegbespalov — DISMISSED** [review]
LGTM!

image

File: lib/consts/consts.go [hunk 1]
```diff
@@ -55,11 +55,11 @@ func FullVersion() string {
 // Banner returns the ASCII-art banner with the k6 logo and stylized website URL
 func Banner() string {
 	banner := strings.Join([]string{
-		`          /\      |‾‾| /‾‾/   /‾‾/   `,
```
Comment (olegbespalov) [inline lib/consts/consts.go:58]: nit: maybe clean the comment :sweat_smile: since as usual it no longer relevant

**Comment by joanlopez** [comment]
image
LGTM!

**Review by joanlopez — DISMISSED** [review]

File: lib/consts/consts.go [hunk 2]
```diff
@@ -55,11 +55,11 @@ func FullVersion() string {
...
-		`     /\  /  \     |  |/  /   /  /    `,
-		`    /  \/    \    |     (   /   ‾‾\  `,
-		`   /          \   |  |\  \ |  (‾)  | `,
-		`  / __________ \  |__| \__\ \_____/ .io`,
+		`         /\       Grafana  /‾‾/  `,
```
Comment (federicotdn) [inline lib/consts/consts.go:58]:
```suggestion
		`         /\      Grafana   /‾‾/  `,
```

**Review by federicotdn — COMMENTED** [review]
I would move the 'Grafana' one char back so that it aligns with the 'k' :)

**Comment by mstoykov** [comment]
@joanlopez does that happen for the previous version as well?

**Review by olegbespalov — APPROVED** [review]

**Comment by joanlopez** [comment]
> @joanlopez does that happen for the previous version as well?

Yeah, yeah. I just mentioned it to clarify the screenshot, as expected 🙏🏻

**Review by joanlopez — APPROVED** [review]

**Pull request merged**